### DISCUSSION TITLE
请问，有没有notepad++ 或 sublime 或 vscode 的自动排版插件？

### DISCUSSION THREAD

**Discussion by excitedbrent** [sparanoid/chinese-copywriting-guidelines#149] [Q&A] [answered]
请问，有没有notepad++ 或 sublime 或 vscode 的自动排版插件？

**Comment by KissMyLady** [comment]
暂时还没有。 sublime代码有自动排版吧

**Comment by cxc654321** [comment]
可能得等等......

**Comment by sparanoid** [comment, marked as answer]
VS Code now has an extension for that: https://github.com/sparanoid/chinese-copywriting-guidelines/pull/131
